OutOfSchool/OutOfSchool.WebApi.Tests/OutOfSchool.WebApi.Tests.csproj

```diff
@@ -31,6 +31,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\OutOfSchool.AuthCommon\OutOfSchool.AuthCommon.csproj" />
+    <ProjectReference Include="..\OutOfSchool.Common\OutOfSchool.Common.csproj" />
```

Comment:
Common project is a transitive dependency of both WebAPi and AuthCommon, it may not be required to add it explicitly